multem2mod/DOI_10_1103_PhysRevB_95_195406_comparison.py

```diff
@@ -0,0 +1,148 @@
+# ------------------------------------------------------------------------------------------------------------------------
```

Comment:
Please, move this file to 'examples' folder

Reply:
It was moved to amos-try/multem2mod/examples/multipole_expansion_verification